src/System.Private.Xml/tests/XmlSerializer/XmlSerializerTests.cs

```diff
@@ -3285,6 +3285,7 @@ public static void Xml_HiddenDerivedFieldTest()
         Assert.Equal(value.value, ((DerivedClass)actual).value);
     }
 
+#if !uapaot
     [Fact]
     public static void Xml_DefaultValueAttributeSetToNaNTest()
```

Comment:
The test failed to build on UWP. This is being tracked by https://github.com/dotnet/corefx/issues/19833.